.github/workflows/debugging.yml

```
@@ -0,0 +1,57 @@
+name: Debugging mops install
+
+on:
+  - pull_request
+
+jobs:
+  run-debugging-steps-v3:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.9
```

```suggestion
        python-version: `3.9`
```
would this make a difference?